Core/GDCore/Project/ObjectGroup.h

```diff
@@ -28,6 +31,18 @@ class GD_CORE_API ObjectGroup {
   ObjectGroup(){};
   virtual ~ObjectGroup(){};
 
+  /**
+   * Must return a pointer to a copy of the object. A such method is needed to
+   * do polymorphic copies. Just redefine this method in your derived object
+   * class like this:
+   * \code
+   * return gd::make_unique<MyObject>(*this);
+   * \endcode
+   */
+  virtual std::unique_ptr<gd::ObjectGroup> Clone() const {
+    return gd::make_unique<gd::ObjectGroup>(*this);

```

Comment:
You don't need Clone I think, because there is no child class so no need for a code to do a `Clone` on an object that could be a child class of this.